package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "pwa-kit",
-  "version": "3.1.0",
-  "lockfileVersion": 3,
```

Comment:
Is there a reason why are we changing the lockfileVersion?

Node 18 ships NPM 9.

The default lockfileVersion in NPM 9 is 3 https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json#lockfileversion